src/reducers/index.js

```diff
@@ -12,9 +12,20 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {
+      ...state,
+      count: state.count+1
```

Comment:
good looking out for pure functionality, right now we only have `count` held on state within our store but if we were to expand this app there might be extra properties held on state and this would make sure they're never affected by this reducer function